stock_move_packaging_qty/models/stock_move_line.py

```diff
@@ -9,11 +10,52 @@ class StockMoveLine(models.Model):
     product_packaging_id = fields.Many2one(
         related="move_id.product_packaging_id", readonly=True
     )
+    product_packaging_qty_reserved = fields.Float(
+        string="Reserved Pkg. Qty.",
+        help="Product packaging quantity reserved.",
+        compute="_compute_product_packaging_qty_reserved",
+        store=True,
+    )
     product_packaging_qty_done = fields.Float(
         string="Done Pkg. Qty.",
         help="Product packaging quantity done.",
+        compute="_compute_product_packaging_qty_done",
+        store=True,
+        readonly=False,
     )
 
+    @api.depends("product_packaging_id", "product_uom_id", "reserved_uom_qty")
+    def _compute_product_packaging_qty_reserved(self):
+        """Get the quantity done in product packaging."""
+        for line in self:
+            if not line.product_packaging_id or not line.product_packaging_id.qty:
+                line.product_packaging_qty_reserved = False
+                continue
+            packaging_uom = line.product_packaging_id.product_uom_id
+            packaging_uom_qty = line.product_uom_id._compute_quantity(
+                line.reserved_uom_qty, packaging_uom
+            )
+            line.product_packaging_qty_reserved = float_round(
+                packaging_uom_qty / line.product_packaging_id.qty,
+                precision_rounding=packaging_uom.rounding,
+            )
```

Comment:
UoM of product_packaging is always the same as product UoM

```suggestion
            line.product_packaging_qty_reserved = float_round(
                line.reserved_uom_qty / line.product_packaging_id.qty,
                precision_rounding=line.product_uom_id.rounding,
            )
```

Reply:
However, it doesn't have to be the same as the UoM in the `stock.move.line`, right? That is what he's handling here AFAICS.